src/config.ts

```diff
@@ -3017,4 +3023,46 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/provenance/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.galaxychain.zone',
```

Comment:
I think wss needs to be enabled
![image](https://user-images.githubusercontent.com/95667791/167033824-ff7663de-d85a-4616-8db0-4d51c8ef49e8.png)

Reply:
I going to ping the team. Back to you when it's ok

Reply:
@JeremyParish69 
'wss://rpc.galaxychain.zone/websocket' It works now.

Reply:
no wss error anymore, but I do get this. Looks like it's being blocked for this staging site.
![image](https://user-images.githubusercontent.com/95667791/167051902-5bb9b566-a405-498e-bbf7-9d4f20104953.png)

Reply:
when i deposit glx at frontier.osmosis.zone
The following error occurred 'network error'

tx details:
{
  "chain_id": "galaxy-1",
  "account_number": "4009",
  "sequence": "42",
  "fee": {
    "gas": "130000",
    "amount": [
      {
        "denom": "uglx",
        "amount": "3250"
      }
    ]
  },
  "msgs": [
    {
      "type": "cosmos-sdk/MsgTransfer",
      "value": {
        "source_port": "transfer",
        "source_channel": "channel-0",
        "token": {
          "denom": "uglx",
          "amount": "1000000"
        },
        "sender": "galaxy1yh4py3f0mym63z7v57krfljnac8ls39akaq02m",
        "receiver": "osmo1yh4py3f0mym63z7v57krfljnac8ls39aze4lhm",
        "timeout_height": {
          "revision_number": "1",
          "revision_height": "4284159"
        }
      }
    }
  ],
  "memo": ""
}

Reply:
@JeremyParish69
<img width="886" alt="스크린샷 2022-05-06 오후 8 25 25" src="https://user-images.githubusercontent.com/99040777/167122675-b3e6bf57-bc1b-4395-9a4b-5263d2bf32ae.png">

Reply:
Hi @jinhopkg I've just try deposit on frontier all goes smooth
![image](https://user-images.githubusercontent.com/58729187/167130132-8962a035-93ab-463c-b2f8-22299a676555.png)